Dockerfile

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------
+# First stage, compile application
+# ----------------------------------------------------------------------
+
+FROM node:14 AS builder
+
+WORKDIR /usr/src/app
+
+# Specify Mapbox Token to use
+ARG MAPBOX_TOKEN=""
```

Comment:
We should also include `PUBLIC_PATH` variable here. It's set in webpack config with the default value of `/`.